manual/softlayer-sru-19.3.41.txt

```diff
@@ -96,3 +99,863 @@ for series in $SRU_SERIES; do
     echo "### END $series"
 done
 
+
+### BEGIN SRU Verification output ###
++ SRU_SERIES='xenial bionic'
++ LP_USER=chad.smith
++ '[' 0 -eq 0 ']'
++ PROPOSED_SCRIPT=setup_proposed.sh
++ cat
++ cat
++ cat
++ sshopts=(-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR)
++ for series in $SRU_SERIES
++ echo '### BEGIN xenial'
+### BEGIN xenial
+++ launch-softlayer --image=os:xenial -u sethostname.yaml
+++ awk '/primary ip/{printf "root@%s", $NF}'
+ [is_ready=True status=no-status]
+Destroy instance with:
+  slcli virtual cancel 93954062
++ VM_IP=root@75.126.0.219
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR root@75.126.0.219 -- dpkg-query --show cloud-init
+cloud-init	19.1-1-gbaa47854-0ubuntu1~16.04.1
++ sleep 30
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR root@75.126.0.219 -- cloud-init status --wait --long
+
+status: done
+time: Mon, 09 Dec 2019 18:14:11 +0000
+detail:
+DataSourceIBMCloud [OS-Code/Live /dev/xvdh]
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR root@75.126.0.219 -- cat /run/cloud-init/result.json
+{
+ "v1": {
+  "datasource": "DataSourceIBMCloud [OS-Code/Live /dev/xvdh]",
+  "errors": []
+ }
+}
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR root@75.126.0.219 -- grep Trace /var/log/cloud-init.log
++ ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR root@75.126.0.219 -- systemd-analyze
+Bootup is not yet finished. Please try again later.
```

Comment:
This suggests we aren't waiting until boot is complete properly. Is that enough of a concern that we need to re-run these?